packages/mask/src/plugins/dHEDGE/UI/PoolViewDesk.tsx

```diff
@@ -0,0 +1,195 @@
+import { UnknowIcon } from '@masknet/icons'
+import { useRemoteControlledDialog, useStylesExtends } from '@masknet/shared'
+import { makeStyles } from '@masknet/theme'
+import { formatBalance } from '@masknet/web3-shared-evm'
+import { Avatar, Button, Link, Paper, Typography } from '@mui/material'
+import BigNumber from 'bignumber.js'
+import { useCallback } from 'react'
+import { Trans } from 'react-i18next'
+import { useI18N } from '../../../utils'
+import { useAvatar } from '../hooks/useManager'
+import { useRewards } from '../hooks/useReward'
+import { PluginDHedgeMessages } from '../messages'
+import type { Pool } from '../types'
+import { formatAmountPostfix } from '../utils'
+import { PerformanceChart } from './PerformanceChart'
+
+const DIGIT_LENGTH = 18
+
+const useStyles = makeStyles()((theme) => ({
+    root: {
+        backgroundColor: '#F6F8F8',
+    },
+    header: {
+        display: 'flex',
+        flexDirection: 'row',
+        padding: theme.spacing(1),
+    },
+    icon: {
+        width: 60,
+        height: 60,
+        borderRadius: '100%',
+        padding: 4,
+        backgroundColor: 'black',
+        justifyContent: 'center',
+        alignItems: 'center',
+        marginRight: theme.spacing(0.5),
+        marginTop: 'auto',
+        marginBottom: 'auto',
+    },
+    avatar: {
+        boardeRadius: '100%',
+        width: '100%',
+        height: '100%',
+    },
+    name: {
+        flex: 1,
+    },
+    buttons: {
+        '>:first-child': {
+            marginRight: theme.spacing(1),
+        },
+        marginTop: 'auto',
+        marginBottom: 'auto',
+    },
+    deck: {
+        paddingLeft: theme.spacing(1),
+        paddingRight: theme.spacing(1),
+        marginTop: theme.spacing(1),
+        '>:first-child': {
+            marginBottom: theme.spacing(1),
+        },
+    },
+    line: {
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+    },
+    text: {
+        flex: 1,
+    },
+}))
+interface PoolViewDeskProps extends withClasses<'root'> {
+    pool: Pool
+    tokens?: string[]
+    link: string
+}
+export function PoolViewDesk(props: PoolViewDeskProps) {
+    const { pool, tokens, link } = props
+    const { t } = useI18N()
+    const classes = useStylesExtends(useStyles(), props)
+
+    //avatar
+    const blockie = useAvatar(pool?.managerAddress ?? '')
+    const valueManaged = formatAmountPostfix(formatBalance(pool?.totalValue ?? 0, DIGIT_LENGTH))
+    const lifeTimeReturn = new BigNumber(formatBalance(pool?.performance ?? 0, DIGIT_LENGTH))
+        .minus(1)
+        .multipliedBy(100)
+        .toFixed(2)
+    const riskFactor = pool && pool?.riskFactor !== -1 ? pool?.riskFactor : '-'
+    const reward = useRewards()
+
+    //#region the invest dialog
```

Comment:
If you use `#region`, you should also provide `#endregion`, which tells where it should end.